baseline/server.js

```diff
@@ -1,268 +1,297 @@
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
+
+let emptyReport = {
+  anonymousSubmission: null,
+  complaintDetailComments: null,
+  completionStepName: null,
+  completionStepNumber: null,
+  createdDate: 1565973780685,
+  crimeType: null,
+  filer: {
+    addresses: [
+      {
+        city: null,
+        country: null,
+        internationalProvinceState: null,
+        poBox: null,
+        postalCode: null,
+        provinceState: null,
+        streetAddress: null,
+        streetName: null,
+        streetNum: null,
+        streetUnit: null,
+      },
+    ],
+    businessName: null,
+    businessOrganizationType: null,
+    emails: [{ address: null }],
+    g1: null,
+    g2: null,
+    languagePreferred: null,
+    languagesSpoken: [],
+    otherName: null,
+    phoneNumbers: [
+      { cleanNumber: null, extension: null, number: null, phoneType: null },
+    ],
+    surname: null,
+    willingToBeContacted: null,
+    isAnonymous: null,
+    isVictim: null,
+    relationshipToVictim: null,
+  },
+  id: 376163,
+  lastModDate: 1565973780716,
+  occurrenceDate: null,
+  paymentMethods: [],
+  personalInfoUsedFor: [],
+  publicFileAttachments: [],
+  publicUser: {
+    id: 195384,
+    isAcceptedDisclaimer: true,
+    isAuthenticated: null,
+    publicComplaint: null,
+  },
+  referenceNumber: '2019-42827',
+  started: true,
+  subcrimeType: null,
+  submissionType: null,
+  summaryComments: null,
+  suspectInfoUsedTypes: [],
+  suspects: [
+    {
+      addresses: [],
+      ageRange: null,
+      businessName: null,
+      businessOrganizationType: null,
+      emails: [],
+      g1: null,
+      g2: null,
+      gender: null,
+      ipAddresses: [],
+      languages: [],
+      orderIndicator: null,
+      phoneNumbers: [],
+      surname: null,
+      websites: [],
+    },
+  ],
+  totalAmountLost: 0.0,
+  totalAmountLostCurrency: null,
+  totalAmountPromised: 0.0,
+  totalAmountPromisedCurrency: null,
+  totalAmountRequested: 0.0,
+  totalAmountRequestedCurrency: null,
+  victim: {
+    addresses: [
+      {
+        city: null,
+        country: null,
+        internationalProvinceState: null,
+        poBox: null,
+        postalCode: null,
+        provinceState: null,
+        streetAddress: null,
+        streetName: null,
+        streetNum: null,
+        streetUnit: null,
+      },
+    ],
+    businessName: null,
+    businessOrganizationType: null,
+    emails: [{ address: null }],
+    g1: null,
+    g2: null,
+    languagePreferred: null,
+    languagesSpoken: [],
+    otherName: null,
+    phoneNumbers: [
+      { cleanNumber: null, extension: null, number: null, phoneType: null },
+      { cleanNumber: null, extension: null, number: null, phoneType: null },
+    ],
+    surname: null,
+    willingToBeContacted: null,
+    ageRange: null,
+    dob: null,
+    gender: null,
+    isAnonymous: null,
+    isBusiness: null,
+  },
+  suspectVictimContactType: null,
+}
+
+let report = {
```

Comment:
Feels like this could be cloned via a deep-copy method or serialized/deserialized from the empty report.

Reply:
Next PR!